### DISCUSSION TITLE
How to remove calendar icon from single post when using minimal-mistakes as a remote github theme?

### DISCUSSION THREAD

**Discussion by Rainymood** [mmistakes/minimal-mistakes#2739] [Q&A] [answered]
Hi,

First of all a big thank you for the theme. I've been using it for years. I have a quick question though:

**How to remove calendar icon from single post when using minimal-mistakes as a remote github theme?**

e.g. I want to remove this icon

![image](https://user-images.githubusercontent.com/2186137/99679141-e8340780-2a7b-11eb-911e-30be9784abe1.png)

I found [this](https://www.cross-validated.com/Personal-website-with-Minimal-Mistakes-Jekyll-Theme-HOWTO-Part-II/#how-to-delete-calendar-icon-in-posts-date) website with the following instructions

- Navigate to `single.html`
- Find `class="page__date"`
- And delete: `<i class="fas fa-fw fa-calendar-alt" aria-hidden="true">`

The problem is that I don't have access to `single.html` because I am using minimal mistakes as a remote github theme.

I am aware that one can overwrite some settings in `assets/css/main.scss` but I don't feel like that will help me here further.

Kind regards,
Jan Meppe

**Comment by iBug** [comment, marked as answer]
If you add the following overrides in your `assets/css/main.scss`, you can get rid of the icon without touching `single.html`:

```scss
.page__meta-date i.far {
  display: none;
}
```

You can refer to [the documentation](https://mmistakes.github.io/minimal-mistakes/docs/stylesheets/#customizing) on customizing CSS.

**Comment by mmistakes** [comment]
You can also create a local copy of [`/_includes/page__meta.html`](https://github.com/mmistakes/minimal-mistakes/blob/master/_includes/page__meta.html) place it in your repo, then remove the calendar icon. Local versions of files will override the file in the remote theme.

**Comment by Rainymood** [comment]
**SOLVED (see edit)**

![image](https://user-images.githubusercontent.com/2186137/99773692-1ada1080-2b0d-11eb-820d-ef6dc369c78a.png)

Thank you for the (very) fast response. I managed to overwrite by copying `/_includes/page__meta.html`. However, this also removes the metadata from each blog post (instead of just the archive). Are these two inextricably linked together or would it be possible to *hide* the date in the archive view but *show* the date on top of each blog post? 

Kind regards, Jan Meppe

EDIT:

How to *hide* the date in each archive view but *show* the date on top of each blog post: 

* copy `_includes/archive-single.html`
* delete line ` {% include page__meta.html type=include.type %}` 
* voila

**Comment by iBug** [comment]
Use my CSS approach so that you can hide icons from very specific places using CSS selectors.

**Comment by Rainymood** [comment]
@iBug so can I conclude that there are 2 ways of effectively changing styles in minimal mistakes? 

* overwriting `_includes` with local copies
* changing `main.scss`

**Comment by iBug** [comment]
Yes. And you need only one of the two methods. The CSS approach is always recommended for easier maintenance and upgrade compatibility, but comes with slight complexity as it requires CSS knowledge.
